### PR TITLE
Hayt should now say what the random vote is.

### PR DESCRIPTION
**Pull request by drobati**
```
hubot vote random
devhost: Your vote totally counted with a yes.
```
or 
```
hubot vote random
devhost: Your vote totally counted with a no.
```

### CODE REVIEW

**Comment by drobati** [comment]
WIP, writing tests.

**Comment by sshirokov** [comment]
What was that? I can't quite hear you

wat

**Comment by drobati** [comment]
@sshirokov Fixed description. :baby_bottle:

File: scripts/vote.coffee [hunk 1]
```diff
@@ -156,7 +156,7 @@ Supported commands:
                 else
                   Vote.current?.no msg.message.user.name
        reply = "Your vote " +
-        (if result then "totally counted." else "was absolutely worthless!")
+        (if result then "totally counted with a #{voteResult}." else "was absolutely worthless!")
```
Comment (sshirokov) [inline scripts/vote.coffee]: Maybe `as a` rather than `with a`?

Sounds more natural to me.

File: tests/vote_test.coffee [hunk 2]
```diff
@@ -14,3 +18,29 @@ describe 'user voting', ->
...
+      
+  context 'vote random', ->
+    it 'should submit a random vote', ->
+      room.user.say 'alice', '@hubot vote poop'
+      room.user.say 'alice', '@hubot vote random'
+      room.user.say 'tom', '@hubot vote random'
+      room.user.say 'sally', '@hubot vote random'
+      room.user.say 'eric', '@hubot vote random'
+      room.user.say 'betsy', '@hubot vote random'
+      sleep 60000 # 60 seconds
```
Comment (sshirokov) [inline tests/vote_test.coffee]: This should literally never appear in tests :(

**Comment by sshirokov** [comment]
Closing because of upstream :rage: delete

**Pull request closed without merging**